index.js

```diff
@@ -16,7 +17,8 @@ if (createId) {
   })
 } else if (applyIds) {
   console.log(c.yellow('applying', applyIds))
-  stack.apply(applyIds, end)
+  var packager = useYarn ? 'yarn' : 'npm'
+  stack.apply(applyIds, end, packager)
```

Comment:
Node convention is to always have the callback as last parameter. You can also remove the `packager` variable, just pass it in directly like:

```js
stack.apply(applyIds, useYarn ? 'yarn' : 'npm', end)
```